docs/.vuepress/theme/components/Footer.vue

```diff
@@ -96,65 +58,232 @@ export default {
     lastUpdatedText() {
       return translate('website-last-updated', this.$lang)
     }
+  },
+  data: function() {
+    return {
+      socialLinks: [
+        {
+          icon: 'github',
+          to: 'https://github.com/ethereum'
+        },
+        {
+          icon: 'twitter',
+          to: 'https://twitter.com/ethereum'
+        },
+        {
+          icon: 'youtube',
+          to: 'https://youtube.com/channel/UCNOfzGXD_C9YMYmnefmPH0g'
+        }
+      ],
+      links: [
+        {
+          title: 'Individuals',
+          items: [
+            {
+              to: '/what-is-ethereum/',
+              text: 'What is Ethereum?',
+              useRouter: true
+            },
+            {
+              to: '/eth/',
+              text: 'What is Ether (ETH)?',
+              useRouter: true
+            },
+            {
+              to: '/dapps/',
+              text: 'Use Ethereum',
+              useRouter: true
+            },
+            {
+              to: '/wallets/',
+              text: 'Ethereum Wallets',
+              useRouter: true
+            },
+            {
+              to: '/learn/',
+              text: 'Guides & Resources',
+              useRouter: true
+            },
+            {
+              to: '/community/',
+              text: 'Ethereum Community',
+              useRouter: true
+            }
+          ]
+        },
+        {
+          title: 'Developers',
+          items: [
+            {
+              to: '/build/',
+              text: 'Get Started',
+              useRouter: true
+            },
+            {
+              to: 'https://studio.ethereum.org/',
+              text: 'Ethereum Studio',
+              newTab: true
+            },
+            {
+              to: '/developers/',
+              text: 'Developer Resources',
+              useRouter: true
+            }
+          ]
+        },
+        {
+          title: 'Ecosystem',
+          items: [
+            {
+              to: 'https://blog.ethereum.org/',
+              text: 'Blog',
+              newTab: true
+            },
+            {
+              to: 'https://ecosystem.support',
+              text: 'Ecosystem Support Program',
+              newTab: true
+            },
+            {
+              to: 'https://eips.ethereum.org/',
+              text: 'EIPs',
+              newTab: true
+            },
+            {
+              to: '/assets/',
+              text: 'Ethereum Brand Assets',
+              useRouter: true
+            }
+          ]
+        },
+        {
+          title: 'About ethereum.org',
+          items: [
```

Comment:
These are incorrect. Let's update these to:
- Language Support
- Privacy Policy
- Terms of Use
- Cookie Policy
- Contact

These links will be the same as there are in current footer

Reply:
whoops!